src/connection.js

```diff
@@ -1149,6 +1150,55 @@ export class Connection {
     };
   }
 
+  /**
+   * Fetch the contents of a Nonce account from the cluster
+   */
+  async getNonceAndContext(
+    nonceAccount: PublicKey,
+    commitment: ?Commitment,
+  ): Promise<RpcResponseAndContext<NonceAccount>> {
+    const args = this._argsWithCommitment(
+      [nonceAccount.toBase58()],
+      commitment,
+    );
+    const unsafeRes = await this._rpcRequest('getAccountInfo', args);
+    const res = GetAccountInfoAndContextRpcResult(unsafeRes);
+    if (res.error) {
+      throw new Error(res.error.message);
+    }
+    assert(typeof res.result !== 'undefined');
+
+    const isV021 =
+      typeof res.result.context !== 'undefined' &&
+      typeof res.result.value !== 'undefined';
+
+    const slot = isV021 ? res.result.context.slot : NaN;
+    const resultValue = isV021 ? res.result.value : res.result;
+
+    if (!resultValue) {
+      throw new Error('Invalid request');
+    }
+
+    const value = NonceAccount.fromAccountData(Buffer.from(resultValue.data));
+
+    return {
+      context: {
+        slot,
+      },
+      value,
+    };
+  }
+  async getNonce(
```

Comment:
whitespace before new function, and esdoc header please

Reply:
I was following the pattern of all the other `AndContext` method pairs on this.
I'll fix this up for all methods it that aforementioned separate PR.